### ISSUE TITLE
Accessibility: Take out extra messages from Notebook verbosity FALSE

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#184173]
CC @meganrogge

Regardless of the Notebook verbosity setting, the following message is not taken out:

* "the inner code editor is focused"

* "the cell output is focused"

`settings.json`:

``` json
{
  "accessibility.verbosity.notebook": false
}
```

## System Info

Version: 1.79.0-insider (system setup)
Commit: 05ae089c791aa5c8ec05abd5687c9a31517579bc
Date: 2023-06-02T09:39:34.473Z
Electron: 22.5.5
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621

**Comment by meganrogge** [comment]
@rebornix I wonder if this is because of the extra `()` you added?

**Comment by meganrogge** [comment]
@jooyoungseo those have always been read by the screen reader. the part to which the accessibility verbosity setting applies is: ` press escape to focus the cell container` etc

**Comment by meganrogge** [comment]
we can discuss excluding the `ariaLabel` entirely though if that would be preferable. here's where it's created

https://github.com/microsoft/vscode/blob/1899f626fdca44ff80c34ac0f0fe13fc0d3d0856/src/vs/workbench/contrib/notebook/browser/notebookEditorWidget.ts#L2230-L2251

**Comment by meganrogge** [comment]
@amunger we can put this info in the help menu so that it doesn't have to be announced every time something is focused.

**Comment by amunger** [comment]
yes, moving that info out of the aria alert should also resolve the remaining issue about image alt text being interrupted.

**Comment by amunger** [comment]
removed in https://github.com/microsoft/vscode/pull/186229
